stories/assets/js/author_filter.js

```diff
@@ -0,0 +1,14 @@
+export const authorFilter = () => {
+  $('.sort-btn').click(function sortclick(event) {
+    event.preventDefault();
+    $('.author-filter').addClass('author-filter-show');
+    $('.close-btn').addClass('show-close');
+    $(this).addClass('hide-sort');
+  });
+  $('.close-btn').click(function closeclick(event) {
+    event.preventDefault();
+    $('.author-filter').removeClass('author-filter-show');
+    $(this).removeClass('show-close');
+    $('.sort-btn').removeClass('hide-sort');
+  });
+};
```

Comment:
@Ashish-srij Change the layout here to use only one single button for toggle and use appropriate jQuery methods to toggle the text name on the button. Something like the following example:
https://codepen.io/shishircoolin/pen/rNYavxK
Also kindly reduce the number of nested divs used to create the structure. Try to create simpler structure.

Reply:
@shishirphadke Done.
I have not used .text toggle on query because it creates problem on switch the language.